lib/kafka-consumer.js

```diff
@@ -124,15 +125,24 @@ function KafkaConsumer(conf, topicConf) {
   this.globalConfig = conf;
   this.topicConfig = topicConf;
 
-  this._consumeTimeout = 1000;
+  this._consumeTimeout = DEFAULT_CONSUME_TIME_OUT;
+  this._retryReadInterval = DEFAULT_RETRY_READ_INTERVAL;
 }
 
 /**
  * Set the default consume timeout provided to c++land
  * @param {number} timeoutMs - number of milliseconds to wait for a message to be fetched
  */
 KafkaConsumer.prototype.setDefaultConsumeTimeout = function(timeoutMs) {
-  this._consumeTimeout = timeoutMs;
+  this._consumeTimeout = Number(timeoutMs) || DEFAULT_CONSUME_TIME_OUT;
+};
+
+/**
+ * Set the default value of waiting for the next time to read when current reading get none data.
+ * @param {number} intervalMs - number of milliseconds to sleep before the next reading
+ */
+KafkaConsumer.prototype.setDefaultRetryReadInterval = function(intervalMs) {
+  this._retryReadInterval = Number(intervalMs) || DEFAULT_RETRY_READ_INTERVAL;
```

Comment:
Also I think you should not wrap interval within `Number(..)` because validation is done in c++ code